### PR TITLE
[Fix] Update the code for handling ragged numpy arrays in numpy >= 1.24.0

### PR DESCRIPTION
**Pull request by dbogunowicz**
Response to: https://github.com/neuralmagic/deepsparse/issues/825

Since NumPy version 19.0, one must specify dtype=object when creating an array from "ragged" sequences, otherwise, one receives a warning:

```bash
VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
```
Starting NumpyPy version 1.24.0, this warning turns into an explicit error:

```bash
ValueError: setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimensions. The detected shape was (128,) + inhomogeneous part.
```

This PR adds the code change necessary to remove the error. Afaik, only transformers QA code requires the update.

This fix should also be backward compatible at least dating back to 1.19 (in our requirements we honor numpy>=1.16.3, maybe it is time to bump this version up?)

### CODE REVIEW

**Comment by dbogunowicz** [comment]
also updated numpy version in the setup.py, tested with the 1.17 and it was too old (gave me a bunch of errors that indicated that it was too old for our requirements)

File: setup.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def _parse_requirements_file(file_path):
 
 
 _deps = [
-    "numpy>=1.16.3",
+    "numpy>=1.19.0",
```
Comment (dbogunowicz) [inline setup.py:88]:
```suggestion
    "numpy>=1.16.3",
```

**Review by corey-nm — DISMISSED** [review]
such a beautiful diff 🥇

File: src/deepsparse/transformers/pipelines/token_classification.py [hunk 2]
```diff
@@ -367,7 +367,7 @@ def route_input_to_bucket(
             input_schema.inputs,
             add_special_tokens=True,
             return_tensors="np",
-            padding=False,
+            padding=True,
```
Comment (bfineran) [inline src/deepsparse/transformers/pipelines/token_classification.py:370]: why are we adding padding here? if there's padding we cant extract the true sequence length, no?

**Review by bfineran — CHANGES_REQUESTED** [review]
is this change backwards compatible?

File: src/deepsparse/transformers/pipelines/token_classification.py (hunk 2 above)
Comment (dbogunowicz) [reply src/deepsparse/transformers/pipelines/token_classification.py:370]: True. Let me further investigate the optimal solution to the problem, I have to admit that I was just quickly taking a look at the problem as a "side-project" and found out that more work is needed to understand it and to get the right solution.

**Comment by dbogunowicz** [comment]
Simple code to reproduce the main issue that blocks us from landing this patch:
```python
from transformers import AutoTokenizer

sequences = ['a ' * 10, 'a ' * 35]
tokenizer = AutoTokenizer.from_pretrained('/home/ubuntu/.cache/sparsezoo/59081a43-86c9-441c-980d-83c23dcd8de5/deployment')
tokens = tokenizer(
    sequences,
    add_special_tokens=True,
    return_tensors="np",
    padding=False,
    truncation=False,
)
```
```bash
ValueError: Unable to create tensor, you should probably activate truncation and/or padding with 'padding=True' 'truncation=True' to have batched tensors with the same length. Perhaps your features (`input_ids` in this case) have excessive nesting (inputs type `list` where type `int` is expected).
```

**Pull request closed without merging**